.gitlab-ci.yml

```diff
@@ -205,7 +205,7 @@ build_cpp_linux_no_remotefs:
     DOCKER_DRIVER: overlay2
   script:
     - apt-get -y update
-    - apt-get -y install apt-transport-https ca-certificates gnupg software-properties-common wget bzip2 build-essential xxd
+    - apt-get -y install apt-transport-https ca-certificates gnupg software-properties-common wget bzip2 build-essential xxd libbz2-dev
```

Comment:
Why is this change needed? Nothing else we changed should be related to this part of the build.

Reply:
My bad, I'll revert and start a new branch